keanu-python/build.gradle

```diff
@@ -143,6 +143,6 @@ pythonVersionInfo.mustRunAfter(installPipenv)
 installDependencies.dependsOn(installPipenv)
 generateDocumentation.dependsOn(preparePythonEnvironment)
 test.dependsOn(preparePythonEnvironment)
-test.dependsOn(formatCheck)
 test.dependsOn(pytest)
+check.dependsOn(formatCheck)
```

Comment:
actually we want `formatCheck` to happen earlier, before `test`. When it fails we want to know early.

Reply:
Yeah fair enough will revert.

Reply:
Done.